components/ace-editor.vue

```diff
@@ -74,13 +74,10 @@ export default {
 
   methods: {
     defineTheme() {
-      if (this.theme) {
-        return this.theme;
-      } else {
-        return (
-          this.$store.state.postwoman.settings.THEME_ACE_EDITOR || DEFAULT_THEME
-        );
-      }
+      if (this.theme) return this.theme;
+      return (
+        this.$store.state.postwoman.settings.THEME_ACE_EDITOR || DEFAULT_THEME
+      );
```

Comment:
```js
if () {
  //
}
return ...
```

Having a consistent style across the codebase is really important. What do you think?